packages/io/src/adapter.ts

```diff
@@ -0,0 +1,26 @@
+import KoaRouter from 'koa-router'
+import { CoR } from './cor'
+
+export class KoaRouterAdapter extends KoaRouter {
+  constructor(private readonly cor: CoR) {
+    super()
+  }
+
+  middleware(): KoaRouter.IMiddleware {
```

Comment:
What's the difference between `middleware` and `routes`

Reply:
No difference, just for compatibility with koa